server/models/ProspectModel.py

```diff
@@ -0,0 +1,23 @@
+from app import db 
+
+prospects_tags = db.Table('prospects_tags',
+    db.Column('prospect_id', db.Integer, db.ForeignKey('prospects.id')),
+    db.Column('tag_id', db.Integer, db.ForeignKey('tags.id'))
+)
+
+class ProspectModel(db.Model):
+    __tablename__ = 'prospects'
+
+    id = db.Column(db.Integer, primary_key = True)
+    email = db.Column(db.String(120), unique = True, nullable = False)
```

Comment:
email does not need to be unique - since a prospect part of one owner could have the same email as aonther prospect by another owner